optimum/intel/openvino/modeling_decoder.py

```diff
@@ -322,6 +340,7 @@ def normalized_config(self):
     def compile(self):
         if self.request is None:
             super().compile()
+            self.compiled_model = self.request
             self.request = self.request.create_infer_request()
```

Comment:
why not remove this
```suggestion
```
and use [self.request](https://github.com/huggingface/optimum-intel/blob/ff1d94b5522a2bdda042f02564b76686bf3add6c/optimum/intel/openvino/modeling_base.py#L408) instead of `self.compiled_model` ? (`self.request` doesn't seem to be used anywhere)